### PR TITLE
[Activation] Removed mempool validator tests that aren't implemented

### PR DESCRIPTION
**Pull request by majikandy**
Removed 45 unimplemented tests.

Added them to Issue #2027 so they can be implemented when it is a priority.

### CODE REVIEW

**Comment by bokobza** [comment]
Instead of removing them, can you please add Skip="some reason" or maybe actually implement them?

**Comment by majikandy** [comment]
It is better to remove them

**Comment by majikandy** [comment]
>Imho they should be implemented instead of being removed.

Absolutely agree - which is why an issue is created, so that can be prioritised into appropriate sprint

**Comment by noescape00** [comment]
>Absolutely agree - which is why an issue is created, so that can be prioritised into appropriate sprint

So let's have the issue but close the PR. 
I don't see a reason why they should be removed from the codebase and stored in the issue

**Comment by majikandy** [comment]
YAGNI is the main reason for removing. Worth noting that it is a copy-paste away from going back into the codebase when the time comes to implement them.   It is good that the thought has gone in to these scenarios so we don't want to lose that.

**Comment by bokobza** [comment]
When you skip them, they will appear as skipped in your test window so we'll always be reminded that they need to be done.
Not keen for removing them at all though.

**Comment by bokobza** [comment]
By all means, please create the issue.

**Comment by majikandy** [comment]
>When you skip them, they will appear as skipped in your test window so we'll always be reminded that they need to be done.

No you won't , you'll get used to ignoring them

**Comment by majikandy** [comment]
Issue #2027  already created and contains the exact stubs to paste back in when the time comes.

**Comment by bokobza** [comment]
and how is removing them going to help?

**Comment by majikandy** [comment]
makes you think less when you don't see anything there in the file - so you don't have to interrogate why skipped

**Comment by majikandy** [comment]
and YAGNI - You ain't gonna need it - we might never ever need to implement these tests, so they might sit there skipped forever causing team thinking time over and over and over again.

**Comment by majikandy** [comment]
Note though, I'm totally up for them being implemented, but is it a priority right now over getting back to a working codebase/test suite ?

**Comment by bokobza** [comment]
Cool, maybe we can rediscuss.
Can you please close this PR? YAGNI or not, I don't want to delete them just to put them back next week.

**Comment by majikandy** [comment]
I totally disagree though,  compromise of skip?

**Comment by majikandy** [comment]
final point... Why so scared of deleting them? we have source control

**Comment by bokobza** [comment]
yeah skip is good.
Try it, you'll see it pops off the screen. is nice

**Comment by majikandy** [comment]
I know it does, but drawing attention to them is what make developers spend too much time on them

**Comment by majikandy** [comment]
Skip reason="Not implemented yet" ?

**Comment by bokobza** [comment]
Sounds good

**Comment by majikandy** [comment]
done

File: src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs [hunk 1]
```diff
@@ -19,76 +19,76 @@ public MempoolValidatorTest() : base(KnownNetworks.RegTest)
...
-        [Fact]
+        [Fact(Skip = "Not implemented yet.")]
         public void CalculateModifiedSize_CalcWeightWithTxIns_ReturnsSize()
         {
             // TODO: Test GetTransactionWeight - InputSizes = CalculateModifiedSize
             // Test weight calculation by input nTxSize = 0
         }
 
-        [Fact]
+        [Fact(Skip = "Not implemented yet.")]
```
Comment (majikandy) [inline src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: oops

Comment (majikandy) [reply src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: Is this test below valid??? has a TODO in it.....

File: src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs [hunk 2]
```diff
@@ -19,76 +19,76 @@ public MempoolValidatorTest() : base(KnownNetworks.RegTest)
...
         public void CalculateModifiedSize_PreCalcWeightWithTxIns_ReturnsSize()
         {
             // TODO: Test GetTransactionWeight - InputSizes = CalculateModifiedSize
```
Comment (majikandy) [inline src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: This test ignored too as it has this todo in it - so unknown if this is a valid test or not

Comment (bokobza) [reply src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: Fair enough

**Pull request merged**